src/edu/ucsb/cs56/projects/games/fish_animation/Fish.java

```diff
@@ -75,66 +76,66 @@ public Fish(double x, double y, double width, double height) {
 	currentX = nextX;
 	currentY = nextY;
 	
-        for(double j = 0; j < 100; j += 1.0) {
-            nextX = currentX - (((3.0 / 4.0) * width) / 100.0);
-            nextY = currentY - ((height / 2.0) * (Math.sin(Math.PI * ((j + 1.0) / 100.0)) - Math.sin(Math.PI * ((j - 1.0) / 100.0))));
-            wholeFish.append(new Line2D.Double(currentX, currentY, nextX, nextY), false);
-            currentX = nextX;
-            currentY = nextY;
-        }
-    }
-    
-    /**
-       gets the xLocation of the fish
-    */
-    @Override 
-    public double getXPos() {
-	return xPos;
-    }
-    
-    /**
-       gets the yLocation of the fish
-    */
-    @Override 
-    public double getYPos() {
-        return yPos;
-    }
-    
-    /**
-       X is set in the constructor
-    */
-    @Override 
-    public void setX(double x) {
+				for(double j = 0; j < 100; j += 1.0) {
+						nextX = currentX - (((3.0 / 4.0) * width) / 100.0);
```

Comment:
What's up with this indentation?  It should be one tab per level, not two.

Reply:
For this one, I replaced 3.0 / 4.0 with 0.75, which is clearer and easier to read.

Reply:
Looks like you only changed it in one of two places.

I was talking about the indentation, though.  It looks like you changed the indentation of this entire file to use two tabs per level.  I'm okay with making the indentation consistent, but if you're gonna use tabs, use _one_ tab per level.

Reply:
I'll try to fix it!

Reply:
Does this still not work?